app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -1,3 +1,5 @@
+@import "colors";
```

Comment:
I think so, yes. And it could become tha base for a theming system in the future.